src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8182,13 +8197,13 @@ namespace UnitarySystems {
             if (this->m_HeatingSpeedNum == 1) {
                 this->m_HeatingSpeedRatio = 0.0;
                 this->m_HeatingCycRatio = this->m_EMSOverrideCoilSpeedNumValue - floor(this->m_EMSOverrideCoilSpeedNumValue);
-                if (this->m_HeatingCycRatio == 0) {
+                if (useMaxedSpeed or this->m_HeatingCycRatio == 0) {
```

Comment:
Whoops! or should be ||

Reply:
😢  I've been switching between C++ and Python today.